### DISCUSSION TITLE
Make the camera behave like EC1 when sliding

### DISCUSSION THREAD

**Discussion by DaDude001** [VickyFrenzy/gmod-enhanced-camera-2#22] [Ideas]
I also use the [Quick Slide addon](https://steamcommunity.com/sharedfiles/filedetails/?id=2416989205) and with EC1, the slide looked great.
However, with EC2, [the body clips into the ground or sometimes doesn't even show at all](https://imgur.com/a/rpfnnX0).  Could you find a way to fix or update this?

**Comment by VickyFrenzy** [comment]
Unfortunately I don't have time to fix compatibility issues at the moment. 
One of the EC2 goals was to make the camera height more reliable.
And yes the current implementation is flawed and needs to be completely rewritten.
You may look and follow the GitHub issues #12 and #13 as they are related to your issue. 
But don't expect any progress to be made any time soon unless this project gets direct external help and contributions to its code.

In the meantime since the original EC1 has less overall compatibility issues you may try using it with a simple patch for Lua errors: https://steamcommunity.com/sharedfiles/filedetails/?id=2038309825

I'm sorry I couldn't fulfil expectations with EC2, it has gotten really complicated to work on.
